AndyDorokhin/HW10/extras/ball.py

```diff
@@ -0,0 +1,25 @@
+""" 
+    Create a class Ball. Ball objects should accept one argument for "ball type" when instantiated.
+    If no arguments are given, ball objects should instantiate with a "ball type" of "regular."
+    Examples:
+    ball1 = Ball()
+    ball2 = Ball("super")
+    ball1.ball_type  #=> "regular"
+    ball2.ball_type  #=> "super"
+"""
```

Comment:
good job
look at this code

class Ball():
  def __init__(self, type = "regular"):
    self.ball_type = type